lampod/src/liquidity.rs

```diff
@@ -0,0 +1,35 @@
+use std::sync::Arc;
+
+use lampo_common::keys::LampoKeysManager;
+use lightning_liquidity::{events::Event, LiquidityManager};
+
+use crate::{chain::LampoChainManager, ln::LampoChannel};
+
+type LampoLiquidity =
+    LiquidityManager<Arc<LampoKeysManager>, Arc<LampoChannel>, Arc<LampoChainManager>>;
+
+struct LampoLiquiditySource {
```

Comment:
`LampoLiquiditySource` is a bad name, and it is clear that you copy it somewhere like https://github.com/tnull/ldk-node-hack/blob/fbad621a89b298d9bc7e8a56badddb2094284d2a/src/liquidity.rs#L460 but in Lampo this is `LampoLiquidityManager`

Otherwise, you should justify why and what does mean `Source`

Reply:
This was already updated inside ba24141d32415272030e1c391e680c514f9a2ece